ports/jetlancer/jetlancer/patch/patchscript

```diff
@@ -0,0 +1,148 @@
+    # Define files to delete
+    local files_to_delete=(
+      "./gamedata/Galaxy.dll"
+      "./gamedata/GOG.gml.dll"
+      "./gamedata/gog.ico"
+      "./gamedata/goggame-1489231569.hashdb"
+      "./gamedata/goggame-1489231569.ico"
+      "./gamedata/goggame-1489231569.info"
+      "./gamedata/goggame-galaxyFileList.ini"
+      "./gamedata/goglog.ini"
+      "./gamedata/Jet Lancer.exe"
+      "./gamedata/Launch Jet Lancer.lnk"
+      "./gamedata/support.ico"
+      "./gamedata/unins000.dat"
+      "./gamedata/unins000.exe"
+      "./gamedata/unins000.ini"
+      "./gamedata/unins000.msg"
+      "./gamedata/steam_api.dll"
+    )
+
+    # Delete unnecessary files
+    for file in "${files_to_delete[@]}"; do
+        if [ -f "$file" ]; then
+            rm "$file"
+            echo "Deleted $file"
+        else
+            echo "$file not found"
+        fi
+    done
+
+    # If "gamedata/data.win" exists and matches the checksum of the GOG or Steam versions
+    if [ -f "./gamedata/data.win" ]; then
+        checksum=$(md5sum "./gamedata/data.win" | awk '{print $1}')
+    
+        # Checksum for the GOG version
+        if [ "$checksum" = "0258cadce342712c2ffbd5fc70e0dfd7" ]; then
+            $ESUDO tools/xdelta3 -d -s gamedata/data.win -f ./patch/gogdata.xdelta gamedata/game.droid && \
+            rm gamedata/data.win
+            echo "GOG data.win has been patched"
+        # Checksum for the Steam version
+        elif [ "$checksum" = "86045b6464bb909851634c3e34b2a82e" ]; then
+            $ESUDO tools/xdelta3 -d -s gamedata/data.win -f ./patch/steamdata.xdelta gamedata/game.droid && \
+            rm gamedata/data.win
+ 	        echo "Steam data.win has been patched"
+        else
+            echo "Error: MD5 checksum of data.win does not match any expected version."
+        fi
+    else    
+        echo "Error: Missing data.win in gamedata folder or game has been patched."
+    fi
+
+    # If "gamedata/audiogroup1.dat" exists and matches the checksum of the Steam version
+    if [ -f "./gamedata/audiogroup1.dat" ]; then
+        checksum=$(md5sum "./gamedata/audiogroup1.dat" | awk '{print $1}')
+    
+        # Checksum for the Steam audiogroup1.dat version
+        if [ "$checksum" = "bcadc44e45f8a6caf9b5c5db538ced13" ]; then
+            $ESUDO tools/xdelta3 -d -s gamedata/audiogroup1.dat -f ./patch/steamaudiogroup1.xdelta gamedata/audiogroup1patched.dat && \
+            rm gamedata/audiogroup1.dat
+            mv gamedata/audiogroup1patched.dat gamedata/audiogroup1.dat
+            echo "Steam audiogroup1.dat has been patched."
+        fi
+    else    
+        echo "Error: Missing audiogroup1.dat in gamedata folder or game has been patched."
+    fi
+
+      # Compress audio
+    if [ -f "$PATCHDIR/compress.txt" ]; then
+        echo "Compressing audio. The process will take 5-10 minutes"  > $CUR_TTY
+        mkdir -p "$TMPDIR"
+        ./tools/gmKtool.py -v -m 262144 -b 64 -d "$TMPDIR" "$PATCHDIR/gamedata/game.droid"
```

Comment:
I think verbose level 2 (`-vv`) is better when used with the patcher tool so that the scrolling screen is more "active" :) What do you think ? (UFO 50 uses level 3)
```suggestion
        ./tools/gmKtool.py -vv -m 262144 -b 64 -d "$TMPDIR" "$PATCHDIR/gamedata/game.droid"
```

Reply:
the audiogroup.dat doesn't take too much time in patching... but I guess it is okay